lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "lerna": "2.9.1",
-  "version": "2.0.0-alpha.2",
+  "version": "2.0.0-alpha.3",
```

Comment:
> Why these versions are bumped in this PR instead of using our ocular publish script?

Yes, due to a chicken-and-egg situation I had to publish alpha.3 from this branch so that the draco modules were available on CDN before the tests could pass.

The publish was done with ocular-dev-tools but the commit will be squashed into this PR. Will publish a clean alpha.4 from master once this lands.